pkg/cmd/release/update_versions.go

```diff
@@ -330,6 +326,9 @@ func updateVersions(_ *cobra.Command, _ []string) error {
 	if err := sendPrReport(releasedVersion, prs, smtpPassword); err != nil {
 		return fmt.Errorf("cannot send email: %w", err)
 	}
+	if len(workOnRepoErrors) > 0 {
+		return errors.Join(workOnRepoErrors...)
+	}
```

Comment:
> If this fails for one repo, will the job still show as failed? If not, then it's going to hide the fact that this failed for some of the repos.

Ah, good callout: I'd updated `updateVersions()` to return the `workOnRepoErrors` if any errors occurred during the `workOnRepo` for loop.